tests/common.rs

```diff
@@ -71,15 +71,16 @@ pub async fn create_test_storage(
     max_blob_size: u64,
 ) -> Result<Storage<KeyTest>, String> {
     let path = env::temp_dir().join(dir_name);
+    let ioring = rio::new().expect("create uring");
     let builder = Builder::new()
         .work_dir(&path)
         .blob_file_name_prefix("test")
         .max_blob_size(max_blob_size)
         .max_data_in_blob(100_000)
         .set_filter_config(Default::default())
-        .allow_duplicates();
-    let ioring = rio::new().expect("create uring");
-    let mut storage = builder.build(ioring).unwrap();
+        .allow_duplicates()
+        .enable_aio(ioring);
```

Comment:
Could also provide func to test for support of ioring.

Reply:
ioring is created outside of pearl, so if user can't create rio, he won't be able to enable aio
```
let rio: Result<Rio, Error> = rio::new();
let storage = if let Ok(ioring) = rio {
    builder
        .enable_aio(ioring)
        .build()
} else {
    builder.build()
};
```